src/applications/claims-status/components/MailOrFax.jsx

```diff
@@ -11,7 +11,7 @@ export default function MailOrFax({ onClose }) {
         aria-label="Close this modal"
         onClick={onClose}
       >
-        <i className="far fa-times-circle" />
+        <i className="fas fa-times-circle" />
```

Comment:
I think there was a mixup here. According to the [design channel](https://dsva.slack.com/archives/C0NGDDXME/p1548260326063800), we want to be using the close button in the screenshot below -

![image](https://user-images.githubusercontent.com/1915775/51768695-06629380-20af-11e9-84d6-787883f2edd3.png)

This is as it is in the site now. However, this is the [solid times-circle](https://fontawesome.com/icons/times-circle?style=solid), not the regular (`far`). I think the `far` class was falling back to `fas`, yielding the correct result, but not the right implementation.

Reply:
This is how it is now -

![image](https://user-images.githubusercontent.com/1915775/51769056-fbf4c980-20af-11e9-9266-f33afa6d45e6.png)